File: fish-spots/src/model/dto/CatchDto.ts
export interface CatchDto {
    readonly catchId?: number;
    readonly outingId: number;
    readonly species: string;
    readonly catchLength?: number;
    readonly catchWeight?: number;
    readonly likes: number;
    imageUrl?: string;
    readonly createdAt?: Date;
    readonly lastUpdatedAt?: Date;
}

File: backend/FishSpots/FishSpots.Domain/RequestModels/CatchDto.cs
namespace FishSpots.Domain.RequestModels
{
    public class CatchDto
    {
        public string Species { get; set; }

        public double CatchLength { get; set; }

        public double CatchWeight { get; set; }

        public string? ImageUrl { get; set; }

    }
}
